.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Feedback
+    url: https://github.com/unicode-org/message-format-wg/discussions/categories/feedback
+    about: Comments, criticisms and other user feedback for the MF2 tech preview
```

Are we abbreviating it as MF2, MFv2, MF2.0?  Let's sidestep by spelling it out like in the README PR #308.  I'll update the blog post draft to be consistent, too.

cc @mihnita

```suggestion
    about: Comments, suggestions, and other user feedback for the MessageFormat 2.0 Technical Preview implementation.
```

---

I suggest that we call it *MessageFormat 2* when we talk about the whole initiative that materialized as our WG, and *MessageFormat 2.0* once we release the final spec.

---

I'd avoid acronyms as much as possible.

---

```suggestion
    about: Comments, suggestions and other user feedback for the MessageFormat 2.0 Technical Preview
```